makefile

```diff
@@ -12,16 +12,17 @@ GOPRIVATE          := "github.com/aserto-dev"
 DOCKER_BUILDKIT    := 1
 
 BIN_DIR            := ./bin
+BASE_DIR           := ${PWD}
 EXT_DIR            := ./.ext
```

Comment:
I don't think you need `BASE_DIR` at all and `EXT_DIR` should be
```
EXT_DIR := ${PWD}/.ext
```

Just look at how it's done in other repo. Let's keep things standardized.